packages/selection/src/useSelection.ts

```diff
@@ -51,35 +51,35 @@ export interface IUseSelectionProps<Item> {
   onFocus?: (focusedItem?: Item) => void;
 }
 
-type onFocusCallback<Item> = (item?: Item) => void;
-type onSelectCallback<Item> = (item?: Item) => void;
+type onFocusFn<Item> = (item?: Item) => void;
+type onSelectFn<Item> = (item?: Item) => void;
```

Comment:
Renamed as variable shadowing occurs below (it's redefined later below as a named argument). ESLint gave a helpful warning.